### ISSUE TITLE
is there an option to search for free apps only?

### ISSUE THREAD

**Issue by yaronyosef** [Akdeniz/google-play-crawler#10]
i used https://code.google.com/p/android-market-api/ but it's old and sometimes buggy when trying to download some apps.

this api is great, but missing search for free apps.

**Comment by Fuzion24** [comment]
What do you mean by 'missing search for free apps'

**Comment by Akdeniz** [comment]
Hi @yaronyosef ,

You can check this answer : https://github.com/Akdeniz/google-play-crawler/issues/6#issuecomment-15925197

For example to list free apps under TOOLS category:

**java -jar googleplay.jar -f crawler.conf list TOOLS -s apps_topselling_free**

**Comment by yaronyosef** [comment]
yes, i saw that answer, your solution for listing apps is only for top selling in specific category without a query term.

but i'm looking for a way to do an open search for a term and get all the results fiterd for free downloads.

currently, the results are a mix of paid and free apps.

is there an option like that?

**Comment by Akdeniz** [comment]
No, there is not. Your only chance is searching for a query and filtering returned results afterward.

**Comment by Fuzion24** [comment]
You can do this very nicely interactively from the SBT console, if you'd like :)

**Comment by yaronyosef** [comment]
ok, thank you.
it would be great to have this feature in the future :)

**Comment by Fuzion24** [comment]
It should have been working before, but I bjorked something.. Check out pull #11
